### PR TITLE
Enhancement: Consolidated Docs

### PR DESCRIPTION
**Pull request by brolewis**
This is my attempt to take our old SDLC, trim it up, and make it more accessible to the world.

### CODE REVIEW

**Comment by brolewis** [comment]
Docs published at https://brolewis.github.io/jolly-good-toolbelt/

File: code-management.rst [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+   To prevent accidentally pushing to the upstream repository,
+   update the update push URL to a non-valid URL.
+   We recommend using DISABLE as a visual indicator::
+
+        git remote set-url --push upstream DISABLE
+
+Repository Conventions
+----------------------
+
+* Each repository should have, in the top level directory:
```
Comment (bradsbrown) [inline code-management.rst]: maybe "in the top level directory OR installed as a callable"?

**Review by bradsbrown — APPROVED** [review]
Approved, with one suggestion for a potential wording tweak for clarity?

**Review by ecedmondson — APPROVED** [review]
i am pleased to have finished reading this PR and can't wait to see Lewis' name on the next NYT bestsellers list for his excellent prose

**Pull request merged**